tests/mocha/priceFetch.test.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import axios from 'axios';
+import { geckoEthToUsd } from '../../src/utils/priceFetch';
+
+describe('ETH price fetch', () => {
+  describe('fetch price 1', () => {
```

Comment:
add a test case when error is throw by axios. verify that you will call the stubbed function 5 times.